packages/legacy/core/App/components/misc/QRScanner.tsx

```diff
@@ -31,30 +31,34 @@ const QRScanner: React.FC<Props> = ({ handleCodeScan, error, enableCameraOnError
   const [showErrorDetailsModal, setShowErrorDetailsModal] = useState(false)
   const { t } = useTranslation()
   const { ColorPallet, TextTheme } = useTheme()
-
   const styles = StyleSheet.create({
-    container: {
-      flex: 1,
-    },
     viewFinder: {
+      marginTop: -90,
+      borderWidth: 320,
+      borderColor: 'rgba(0, 0, 0, 0.4)',
+      alignSelf: 'center',
+    },
+    innerFinder: {
+      borderRadius: 6,
+      borderWidth: 2,
+      borderColor: 'rgba(255, 255, 255, 0.6)',
       width: 250,
       height: 250,
-      borderRadius: 24,
-      borderWidth: 2,
-      borderColor: ColorPallet.grayscale.white,
-    },
-    viewFinderContainer: {
-      flexGrow: 1,
-      justifyContent: 'center',
-      alignItems: 'center',
     },
     messageContainer: {
+      zIndex: 999,
```

Comment:
Changing `zIndex` should also be avoided. It affects iOS and Android differently, and the more zIndex changes in the codebase the more likely we are to run into unexpected UI appearance and styling difficulty. In this specific case, I don't see any effect of the zIndex that couldn't be achieved with just altering the order of JSX elements.